addons/account/models/company.py

```diff
@@ -109,7 +109,8 @@ class ResCompany(models.Model):
     account_setup_coa_state = fields.Selection(ONBOARDING_STEP_STATES, string="State of the onboarding charts of account step", default='not_done')
     account_setup_taxes_state = fields.Selection(ONBOARDING_STEP_STATES, string="State of the onboarding Taxes step", default='not_done')
     account_onboarding_invoice_layout_state = fields.Selection(ONBOARDING_STEP_STATES, string="State of the onboarding invoice layout step", default='not_done')
-    account_onboarding_create_invoice_state = fields.Selection(ONBOARDING_STEP_STATES, string="State of the onboarding create invoice step", default='not_done')
+    account_onboarding_create_invoice_state = fields.Selection(ONBOARDING_STEP_STATES, string="State of the onboarding create invoice step", compute='_compute_account_onboarding_create_invoice_state')
+    account_onboarding_create_invoice_state_flag = fields.Boolean(default=False, store=True)
```

Comment:
I don't really see the point of this additional field if it is never set nor used elsewhere than in the compute function of `account_onboarding_create_invoice_state`

Reply:
I think that what you want is setting the field to `True` when calling `action_open_account_onboarding_create_invoice`